mm2src/coins/solana.rs

```diff
@@ -283,21 +281,21 @@ async fn withdraw_impl(coin: SolanaCoin, req: WithdrawRequest) -> WithdrawResult
 
 impl SolanaCoin {
     pub async fn estimate_withdraw_fees(&self) -> Result<(solana_sdk::hash::Hash, u64), MmError<ClientError>> {
-        let hash = self.rpc().get_latest_blockhash().await?;
+        let hash = block_in_place(|| self.rpc().get_latest_blockhash())?;
```

Comment:
As per official `block_in_place` docs https://docs.rs/tokio/latest/tokio/task/fn.block_in_place.html, it doesn't block the executor. The only downside is: 
> Be aware that although this function avoids starving other independently spawned tasks, any other code running concurrently in the same task will be suspended during the call to block_in_place. This can happen e.g. when using the [join!](https://docs.rs/tokio/latest/tokio/macro.join.html) macro. To avoid this issue, use [spawn_blocking](https://docs.rs/tokio/latest/tokio/task/fn.spawn_blocking.html) instead of block_in_place.

So I guess it will block other tasks in RPC batch requests case, so using `async_blocking` is more preferable. Will refactor.

Reply:
Ooh I see, thank you for the note!
> Runs the provided blocking function on the current thread without blocking the executor.